src/app/public/src/modules/wrapper/wrapper.component.spec.ts

```diff
@@ -287,12 +289,50 @@ describe('StacheWrapperComponent', () => {
     expect(mockTitleService.setTitle).toHaveBeenCalledWith('Test Title');
   });
 
-  it('should set the page title', () => {
+  it('should set the window title based on stache pageTitle', () => {
     component.pageTitle = 'Page Title';
     fixture.detectChanges();
     expect(mockTitleService.setTitle).toHaveBeenCalledWith('Page Title');
   });
 
+  it('should set the window title based on stache navTitle', () => {
+    component.navTitle = 'Nav Title';
+    fixture.detectChanges();
+    expect(mockTitleService.setTitle).toHaveBeenCalledWith('Nav Title');
+  });
+
+  it('should set the window title based on the Tutorial Header title', () => {
+    mockTextContent = 'Tutorial Header Title';
+    component.ngOnInit();
+    fixture.detectChanges();
+    expect(mockTitleService.setTitle).toHaveBeenCalledWith('Tutorial Header Title');
+    mockTextContent = '';
+  });
+
+  it('should not set the window title when the Tutorial Header title is absent', () => {
+    fixture.detectChanges();
+    expect(mockTitleService.setTitle).not.toHaveBeenCalledWith('Tutorial Header Title');
```

Comment:
This test doesn't really tell us much except it wasn't called with `Tutorial Header Title`.  What was it called with? Anything?  What gets passed in if there is no pageTitle, navTitle, or tutorial header? etc.